core/bindings.rs

```diff
@@ -17,6 +17,9 @@ use std::option::Option;
 use url::Url;
 use v8::MapFnTo;
 
+use serde::{Serialize};
```

Comment:
I guess I shouldn't complain if clippy doesn't but since we don't use this pattern anywhere else...
```suggestion
use serde::Serialize;
```